official-templates/stable-diffusion-webui/Dockerfile

```diff
@@ -1,29 +1,24 @@
 FROM nvidia/cuda:11.8.0-devel-ubuntu22.04
 
 ARG WEBUI_VERSION
+ARG TORCH_VERSION
+ARG XFORMERS_VERSION
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV SHELL=/bin/bash
 ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib/x86_64-linux-gnu
 ENV PATH="/workspace/venv/bin:$PATH"
 
-# Used by webui.sh
```

Comment:
This is not required because torch is already installed by this Dockerfile and the webui-user.sh script has `--skip-install` enabled by default anyway.